examples/image_embedding/EmbeddingModel_triplet_with_cat_bird_data.ipynb

```diff
@@ -1,5 +1,90 @@
 {
  "cells": [
+  {
```

Comment:
similar comments from above (e.g., the overview section and pretty print of some cells)
+

we need more comments to guide people for the cells [14] through [22]